src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java

```diff
@@ -327,8 +326,7 @@ private JPanel getJCodeLabelPanel() {
 	 */
 	private JPanel getJDescriptionLabelPanel() {
 		if (jDescriptionLabelPanel == null) {
-			jDescriptionLabel = new JLabel();
-			jDescriptionLabel.setText(MessageBundle.getMessage("angal.preagtreattype.description2"));
+			jDescriptionLabel = new JLabel(MessageBundle.getMessage("angal.preagtreattype.description"));
```

Comment:
we can replace with "angal.common.description"?

Reply:
Replaced with new format:  angal.common.description.txt